powerline/config_files/themes/powerline_unicode7.json

```diff
@@ -143,6 +143,15 @@
 			"args": {
 				"text": "🖫⃥"
 			}
+		},
+
+		"powerline.segments.i3wm.scratchpad": {
+			"args": {
+				"icons": {
+					"fresh": "O",
+					"changed": "X"
```

Comment:
Unicode 7.0. UTF-7 is *ASCII only* encoding scheme, it *cannot* produce symbols that Vim cannot render unless Vim parsed file as UTF-7. And many symbols are common with powerline theme.